src/redux/slices/feedSlice.ts

```diff
@@ -16,19 +21,47 @@ interface FeedQuestionType {
     | typeof LOADING
     | typeof ERROR;
   questionError: string;
+  addCommentAPIStatus:
+    | typeof IDLE
+    | typeof FULFILLED
+    | typeof LOADING
+    | typeof ERROR;
```

Comment:
again all these constants are of type string so it will would be better to directly pass it as `string` 

`addCommentAPIStatus: string` 

The other good way to restrict the the value of `addCommentAPIStatus is to directly use the value. Instead  of using `typeof` directly pass the value 

```ta
type StatusType =  IDLE |  FULFILLED | LOADING | ERROR;
```

Correct me if I am missing something.

Reply:
Yes, got it. Will include the changes in the next PR.
Thanks for the review :)